DESCRIPTION

```diff
@@ -3,7 +3,7 @@ Title: What the Package Does (one line, title case)
 Version: 0.0.1
 Authors@R: person("First", "Last", email = "first.last@example.com", role = c("aut", "cre"))
 Description: What the package does (one paragraph).
-Depends: R (>= 3.5.1)
```

Comment:
not a big problem, but for simplicity, might be worth updating your R version on your machine to 3.5 or greater?